Dockerfile

```diff
@@ -1,32 +1,41 @@
-FROM ocaml/opam:alpine-3.20-ocaml-5.2 as build
+FROM ocaml/opam:alpine-3.20-ocaml-5.2 AS build
 
 # Install system dependencies
-RUN sudo apk update && sudo apk add --update libev-dev openssl-dev gmp-dev oniguruma-dev inotify-tools curl-dev autoconf
+RUN apk -U upgrade --no-cache && apk add --no-cache \
```

Comment:
```suggestion
RUN sudo apk -U upgrade --no-cache && sudo apk add --no-cache \
```

Reply:
I removed the `sudo` by mistake. In the base images, the default user is `opam`.